app/Http/Consts/UrlInfo.php

```diff
@@ -1,11 +1,11 @@
 <?php
 
-namespace App\Http;
+namespace App\Http\Consts;
 
 /**
- * syonetに用いるURLやパスを用いるクラス
+ * syonetに用いるURLを用いるクラス
```

Comment:
__[code_sniffer]__
* Opening brace of a class must be on the line after the definition